portfolio/src/main/webapp/script.js

```diff
@@ -49,3 +49,32 @@ function openLinkedInUrl() {
 function openGitHubUrl() { 
     window.open("https://github.com/RossJ24","_blank");
 }
+/**
+ * Fetches a greeting with my name from the server.
+ */
+async function getGreeting(){
+ let res = await fetch('/data');
```

Comment:
you can use promises here instead of blocking calls. Ex. fetch(...).then(response => response.text()).then(response => addGreetingToDom(response));

Reply:
So I reread the javascript style practices at google. It seems that either is fine, however, we cannot mix both. For this project it is fine. When you work with Jalyn and Dangely, please make sure to establish one convention over the other.